docs/docs/adrs/adr-010-standalone-changeover.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 11
+title: Standalone to Consumer Changeover
+---
+## ADR 010: Standalone to Consumer Changeover
+
+## Changelog
+
+* 6/30/23: Feature completed, first draft of ADR.
+
+## Status
+
+Implemented
+
+## Context
+
+[Stride](https://github.com/Stride-Labs/stride) will be the first consumer to "changeover" from a standalone cosmos blockchain, to a consumer chain secured by the Cosmos Hub. This document will outline the changes made to the replicated security protocol to support this changeover process.
+
+## Decision
+
+### Process
+
+Prior to the changeover, the consumer chain will have an existing staking keeper and validator set, these may be referred to as the "standalone staking keeper" and "standalone validator set" respectively.  
+
+The first step in the changeover process is to submit a ConsumerAdditionProposal. If the proposal passes, the provider will create a new IBC client for the consumer at spawn time, with the provider's validator set. A consumer genesis will also be constructed by the provider for validators to query. Within this consumer genesis contains the initial validator set for the consumer to apply after the changeover.
+
+The consumer upgrade height must be reached after the provider has created the new IBC client. Any replicated security validators who will run the consumer, but are not a part of the sovereign validator set, must sync up a full node before the consumer upgrade height is reached. The disc state of said full node will be used to run the consumer chain after the changeover has completed.
```

Comment:
Maybe mention that the consumer chain must perform a chain upgrade to add the CCV consumer module.